packages/roosterjs-editor-dom/lib/utils/adaptFontColorToBackgroundColor.ts

```diff
@@ -0,0 +1,76 @@
+const TRANSPARENT = 'transparent';
+const DARK = 'DARK';
+const BRIGHT = 'BRIGHT';
+const DEFAULT = 'DEFAULT';
+
+/**
+ * Change the font color to white or some other color, so the text can be visible with a darker background
+ * @param element The element that contains text.
+ * @param newColor The font color to be applied. If not defined, the font color will be white.
+ */
+export default function adaptFontColorToBackgroundColor(
+    element: HTMLElement,
+    newFontColorInDark?: string,
+    newFontColorInBright?: string
+) {
+    if (element.hasChildNodes() && element.firstElementChild?.hasAttribute('style')) {
+        return;
+    }
+    const backgroundColor = element.style.backgroundColor;
+    applyFontColor[isBrightOrDark(backgroundColor)](
+        element,
+        newFontColorInDark,
+        newFontColorInBright
+    );
+}
+
+const applyFontColor: Record<
+    string,
+    (element: HTMLElement, newFontColorInDark?: string, newFontColorInBright?: string) => void
+> = {
+    DARK: (element, newFontColorInDark) => (element.style.color = newFontColorInDark || '#ffffff'),
+    BRIGHT: (element, newFontColorInBright) =>
+        (element.style.color = newFontColorInBright || '#000000'),
+    DEFAULT: element => (element.style.color = ''),
+};
+
+function isBrightOrDark(color: string) {
```

Comment:
the function name makes me feel it should only return 2 possible values, but it actually can return 3